MidasCivil_Engine/Convert/ToBHoM/Results/ToBarForce.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Structure.Loads;
+using BH.oM.Structure.Results;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace BH.Engine.MidasCivil
+{
+    public static partial class Convert
+    {
+        public static BarForce ToBarForce(this List<string> delimitted)
+        {
+            double position = 0;
+            if (delimitted[9].Contains('4'))
+            {
+                    position = 1;
+            }
+            else if (delimitted[9].Contains('3'))
+            {
+                position = 0.75;
+            }
+            else if (delimitted[9].Contains('2'))
+            {
+                position = 0.5;
+            }
+            else if (delimitted[9].Contains('1'))
+            {
+                position = 0.25;
+            }
```

Comment:
These should be more explicit i.e.` .Contains("3/4)`. 
You could have a scenario whereby:
`delimitted[9] = I[3]` and your `if `statement would assign it a position of 0.75 where in reality it is 0.

You also have not covered the cases for I[node number] and J[node number].

I would recommend a `switch `statement instead of 6 if statements.

Also, I would put this in a private method as you reuse it in `ToBarStress.cs` too.